pandas/core/ops.py

```diff
@@ -6,8 +6,10 @@
 import datetime
 import operator
 import textwrap
+from typing import Dict, Optional
 import warnings
 
+from mypy_extensions import TypedDict
```

Comment:
We never added mypy_extensions so this won't work as is

Reply:
Though nice idea. I think on track for 3.8 so maybe just add a comment

Reply:
Well that's embarrassing. I would have sworn it got added, but I'm clearly imagining things.

> Though nice idea. I think on track for 3.8 so maybe just add a comment
 Do you mean comment out the `Operator_description` with a note to use if/when `TypedDict` is available?

Presumably there's another approach that will work meantime (or instead).

Reply:
Open to whatever you think is best